charts/prometheus-stackdriver-exporter/templates/deployment.yaml

```diff
@@ -73,10 +73,10 @@ spec:
               mountPath: /etc/secrets/service-account/
           {{- end}}
           args:
-            - --google.project-id={{ .Values.stackdriver.projectId }}
```

Comment:
can you make this conditional on appVersion. otherwiese it will be a breakting change.

Reply:
old flag is still supported, new flag is **repeatable**